feat(runtime): add info about connector failure retry change (#1905)
bonitasoft/bonita-doc: modules/ROOT/pages/release-notes.adoc

* in the <BONITA>/server/conf/tomcat-users.xml file in a *Bonita Tomcat bundle*
* using xref:bonita-docker-installation.adoc#_monitoring_username[specific Environment parameters] when starting a *Bonita Docker image*

=== Other changes

> * When the engine fails to evaluate output operations of a connector, the connector is no more automatically retried. It is put as failed to avoid automatic re-execution.

Reviewer: We should explain that the decision is for the Customer to make, to re-execute it or not.
And precise the impacts of re-executing it.

> * When the engine fails to evaluate output operations of a connector, the connector is no more automatically retried. It is put as failed to avoid automatic re-execution.

Author: ```suggestion
* When the engine fails to evaluate output operations of a connector, the connector is no more automatically retried. It could be an issue to re-execute a connector because, connectors might have side effects. It is the choice of the administrator of the platform to re-execute the connector or not. See xref:runtime:connectors-execution.adoc[documentation about connector execution] for details about connector failure and replay.
```

> * When the engine fails to evaluate output operations of a connector, the connector is no more automatically retried. It could be an issue to re-execute a connector because, connectors might have side effects. It is the choice of the administrator of the platform to re-execute the connector or not. See xref:runtime:connectors-execution.adoc[documentation about connector execution] for details about connector failure and replay.

Author: ```suggestion
* When the engine fails to evaluate output operations of a connector, the connector is no more automatically retried. It could be an issue to re-execute a connector because connectors might have side effects. It is the choice of the administrator of the platform to re-execute the connector or not. See xref:runtime:connectors-execution.adoc[documentation about connector execution] for details about connector failure and replay.
```

== Bug fixes

> ===
> 

Author: ```suggestion
```
